### PR TITLE
Class Skill Point Clarity

### PR DESCRIPTION
**Pull request by rosebrallier**
-Deleted extraneous note about gaining skills via class
-Changed structure of skill point grants via class
-Deleted a bunch of unnecessary periods

### CODE REVIEW

**Comment by rosebrallier** [comment]
Per Issue #675

File: CharacterCreation/02_Classes.txt [hunk 1]
```diff
@@ -112,9 +107,9 @@ Class Primary Attribute: Strength
 Starting Wealth: $4150 + 100*Luck
 
 Passives: 
-* Gain Skill Weapons - Thrown OR Weapons - Explosives at 10.
-* Gain Skill Weapons - Heavy at 10.
-* Gain Feat: Proficiency: Light Melee Weaponry.
+* Gain  10 skill points in Weapons - Thrown OR Weapons - Explosives
```
Comment (Turtlelord26) [inline CharacterCreation/02_Classes.txt]: Every class after soldier has an extra space in these lines between Gain and 10.

**Review by Turtlelord26 — APPROVED** [review]
Excellent
🐢

File: CharacterCreation/02_Classes.txt (hunk 1 above)
Comment (rosebrallier) [reply CharacterCreation/02_Classes.txt]: The joys of copy-pasta. will fix

**Pull request merged**